src/config.rs

```diff
@@ -2570,6 +2573,7 @@ impl TiKvConfig {
                     + self.raftdb.defaultcf.block_cache_size.0,
             });
         }
+        self.backup.sst_max_size = self.coprocessor.region_max_size;
```

Comment:
Looks like we can adjust the sst_max_size via config file? It always be set to `region_max_size`?